### ISSUE TITLE
highpass filter from edf may be incorrect

### ISSUE THREAD

**Issue by robertoostenveld** [mne-tools/mne-python#3095]
io/edf.py line 103 reads

```
                info['highpass'] = float(np.min(highpass))
                warnings.warn('Channels contain different highpass filters. '
                              'Highest filter setting will be stored.')
```

which returns the lowest, not the highest filter setting. A few lines down it is correctly handled for the lowpass filter.

**Comment by teonbrooks** [comment]
@robertoostenveld thanks for pointing this out.

**Comment by teonbrooks** [comment]
will be resolved with #3096
